testing/cypress/integration/myaccount/myaccount.spec.js

```diff
@@ -0,0 +1,142 @@
+
+        cy.get(".notification-wrapper").children().first().should("contain", "Eine E-Mail zur Bestätigung der Änderungen wurde an Ihre E-Mail-Adresse gesendet.");
+
+        cy.wait("@resetMail").its("response.statusCode").should("eq", 200);
+    });
+
+    it("should NOT change mail address if it already exists", () =>
+    {
+        cy.login("stefan.standard@plentye2etest.de");
+        cy.visit("/myaccount");
+
+        cy.get(".widget-account-settings").find(".add-item").children().first().click();
+        cy.intercept("POST", "/rest/io/customer/mail/").as("resetMail");
+        cy.getByTestingAttr("change-mail").type("bernd.business@plentye2etest.de", { delay: 40 });
+        cy.getByTestingAttr("change-mail-repeat").type("bernd.business@plentye2etest.de", { delay: 40 });
+        cy.getByTestingAttr("change-mail-submit").click();
+
+        cy.get(".notification-wrapper").children().first().should("contain", "Für diese E-Mail-Adresse existiert bereits ein Konto.");
+        cy.wait("@resetMail").its("response.statusCode").should("eq", 400);
+    });
+
+    it("should change password address", () =>
+    {
+        cy.login("stefan.standard@plentye2etest.de");
+        cy.visit("/myaccount");
+
+        cy.intercept("POST", "/rest/io/customer/password/").as("resetPassword");
+        cy.get(".widget-account-settings").find(".add-item").children().last().click();
+        cy.getByTestingAttr("old-password").type("Testuser1234", { delay: 40 });
+        cy.getByTestingAttr("new-password").type("Testuser1234", { delay: 40 });
+        cy.getByTestingAttr("new-password-repeat").focus();
+        cy.getByTestingAttr("new-password-repeat").type("Testuser1234", { delay: 40 });
+        cy.getByTestingAttr("new-password-submit").click();
+
+        cy.get(".notification-wrapper").children().first().should("contain", "Das Passwort wurde erfolgreich geändert.");
+        cy.wait("@resetPassword").its("response.statusCode").should("eq", 200);
+    });
+
+    it("should NOT change password address if old password is wrong", () =>
+    {
+        cy.login("stefan.standard@plentye2etest.de");
+        cy.visit("/myaccount");
+
+        cy.intercept("POST", "/rest/io/customer/password/").as("resetPassword");
+        cy.get(".widget-account-settings").find(".add-item").children().last().click();
+        cy.getByTestingAttr("old-password").type("TestuserWrong", { delay: 40 });
+        cy.getByTestingAttr("new-password").type("Testuser1234", { delay: 40 });
+        cy.getByTestingAttr("new-password-repeat").focus();
+        cy.getByTestingAttr("new-password-repeat").type("Testuser1234", { delay: 40 });
+        cy.getByTestingAttr("new-password-submit").click();
+
+        cy.get(".notification-wrapper").children().first().should("contain", "Das Passwort konnte nicht geändert werden.");
+        cy.wait("@resetPassword").its("response.statusCode").should("eq", 401);
+    });
+
+    it("should add new address", () =>
```

Comment:
Ich würde die Address tests auslagern in eine eigene Datei da kommen bestimmt noch mehr dazu.